app/src/main/java/com/example/demo/MainActivity.kt

```diff
@@ -19,6 +19,15 @@ class MainActivity : ComponentActivity() {
         enableEdgeToEdge()
         setContent {
             DemoTheme {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty